generics/scheduler/user-profile.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * name : delete-read-notifications.js
+ * author : Aman Jung Karki
+ * Date : 15-Nov-2019
+ * Description : Delete all read notifications every month for samiksha.
+ */
+
+// dependencies
+
+const kafkaCommunication = require(ROOT_PATH + "/generics/helpers/kafka-communications");
+const slackClient = require(ROOT_PATH + "/generics/helpers/slack-communications");
+const UNNATI_INDEX =
```

Comment:
@aman-tunerlabs - If this is not used can it be removed ?

Reply:
Removed